<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenCVDetection</class>
 <widget class="QMainWindow" name="OpenCVDetection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>407</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>OpenCVDetection</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>341</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_open_image">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_open_model">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_video">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Video</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>FPS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_fps">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>360</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Detect Time</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_detect">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Detect</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>210</y>
      <width>131</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Threshold</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Iou Thresold</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>conf Thresold</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinbox_iou_threshold">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
     <property name="value">
      <double>0.450000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinbox_conf_threshold">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
     <property name="value">
      <double>0.250000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
